day_7/to_words/to_words.gemspec

```diff
@@ -0,0 +1,28 @@
+require_relative 'lib/to_words/version'
```

Comment:
I'm unsure how much config you needed to do to get a gem up and running, but I'm sure that was a learning experience into itself! Another challenge is to not use a gem, and see how key/value pairs can serve as a reference point to change integers into spelled out words.